src/map/clif_packetdb.hpp

```diff
@@ -290,6 +290,7 @@
 	packet(0x01dd,47);
 	packet(0x01de,33);
 	parseable_packet(0x01df,6,clif_parse_GMReqAccountName,2);
+	parseable_packet(0x0a88,2,clif_parse_GMCooldownReset,0);
```

Comment:
```suggestion
#if PACKETVER_MAIN_NUM >= 20160622 || PACKETVER_RE_NUM >= 20160622 || defined(PACKETVER_ZERO)
	parseable_packet(0x0a88,2,clif_parse_GMCooldownReset,0);
#endif
```
Please also move it down. Maybe replace the `2016-06-22aRagexeRE` entry that is not used anymore.